js/views/modals/purchase/Purchase.js

```diff
@@ -105,14 +111,17 @@ export default class extends BaseModal {
       selectFirst: true,
       radioStyle: true,
     });
+    // render the moderators so it can start fetching and adding moderator cards
+    this.moderators.render();
+    this.moderators.getModeratorsByID();
```

Comment:
The moderators view is being rendered here because getModeratorsByID will add the moderators to the DOM, which fails if it isn't rendered yet. We don't want to call that in the render of this view or it would refetch all the moderators on each render.